### PR TITLE
Misc stuff, look at commits

### PR DESCRIPTION
**Pull request by RealKC**
Pray to God I didn't mess up

### CODE REVIEW

**Comment by napstr** [comment]
```
[ERROR] /home/travis/build/Frederikam/FredBoat/FredBoat/src/main/java/fredboat/commandmeta/init/MusicCommandInitializer.java:[84,58] cannot find symbol
  symbol:   class UserInfoCommand
  location: class fredboat.commandmeta.init.MusicCommandInitializer
```

**Comment by RealKC** [comment]
@napstr mergeable now?

File: FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java [hunk 1]
```diff
@@ -38,42 +38,45 @@
...
+        CommandRegistry.registerCommand("hardban", new HardbanCommand());
+        CommandRegistry.registerCommand("kick", new KickCommand());
+        CommandRegistry.registerCommand("softban", new SoftbanCommand());
+        CommandRegistry.registerCommand("clear", new ClearCommand());
+        
+        /* Util */
         CommandRegistry.registerCommand("serverinfo", new fredboat.command.util.ServerInfoCommand(), "guildinfo");
-        CommandRegistry.registerCommand("invite", new InviteCommand());
         CommandRegistry.registerCommand("userinfo", new fredboat.command.util.UserInfoCommand(), "memberinfo");
-        CommandRegistry.registerCommand("gitinfo", new GitInfoCommand(), "git");
```
Comment (napstr) [inline FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java]: this line is deleted but never shows up again

File: FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java [hunk 2]
```diff
@@ -38,42 +38,45 @@
...
-        CommandRegistry.registerCommand("exit", new ExitCommand());
```
Comment (napstr) [inline FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java]: this line is deleted without replacement

**Comment by RealKC** [comment]
What about now?

**Pull request merged**